docs/start/getting-started/fragments/ios/add-api.md

```diff
@@ -21,13 +21,27 @@ Now that you have DataStore persisting data locally, in the next step you'll con
     ```bash
     amplify push
     ```
-> Reply `No` when it asks for `? Do you want to generate code for your newly created GraphQL API`.
+  
+  This command will display a summary of the changes that will be pushed to your backend. It should look like this:
 
-    <amplify-callout>
-    
-    **Note:** sit back and relax since this command will generate all the required cloud resources on your AWS account and it might take a while to complete.
+  ![](~/images/lib/getting-started/ios/amplify-push-api-summary.png)
+
+  <amplify-callout>
+
+  **Note:** If the summary doesn't contain the API category, you will need to cancel this push process and run `$ amplify add api` before continuing.
```

Comment:
I have a few concerns with this, because `amplify add api` is not enough. The developer needs to pick specific options in order for DataStore to work (GraphQL, enable conflict resolution).

I believe the fact that someone ignored the `--quickstart` means we could improve the first steps, but I'm not sure this is the right fix for that.